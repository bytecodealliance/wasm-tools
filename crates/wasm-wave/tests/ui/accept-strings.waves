string("");
string("WAVEğŸŒŠ");
string("\\\'\"\t\n\r");
string("'\'\u{27}");
string("â˜ƒ\u{2603}\u{002603}");
// Multiline: empty
string("""
""");
// Multiline: escapes
string("""
Break up double quote sequences: ""\""
Other escapes work: \\\'\"\t\n\r\u{2603}
""");
// Multiline: indent behavior
string("""
    Trailing indent
  determines dedent
      behavior.
  """);
// Multiline: empty lines
string("""

Empty leading and trailing lines are retained

""")