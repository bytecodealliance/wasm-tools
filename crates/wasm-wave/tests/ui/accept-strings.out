string(val: "")
string(val: "WAVEðŸŒŠ")
string(val: "\\\'\"\t\n\r")
string(val: "\'\'\'")
string(val: "â˜ƒâ˜ƒâ˜ƒ")
// Multiline: empty
string(val: "")
// Multiline: escapes
string(val: "Break up double quote sequences: \"\"\"\"\nOther escapes work: \\\'\"\t\n\râ˜ƒ")
// Multiline: indent behavior
string(val: "  Trailing indent\ndetermines dedent\n    behavior.")
// Multiline: empty lines
string(val: "\nEmpty leading and trailing lines are retained\n")
